src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1299,6 +1299,11 @@ public function renderFieldAction($productId, $step, $fieldName)
     }
 
     /**
+     * @AdminSecurity(
+     *     "is_granted(['read'], request.get('_legacy_controller')) || is_granted(['create', 'update'], 'AdminOrders')",
```

Comment:
This is weird to rely on Orders authorizations to be able to search the product list

Reply:
Agreed, this is mostly because this API is used for Order creation only The problem is more that it should be in another controller I think